File: TEC-KasinoAPI/Models/Customer Models/CustomerRegisterRequest.cs
using System.ComponentModel.DataAnnotations;

namespace TEC_KasinoAPI.Models
{
    public class CustomerRegisterRequest
    {
        [Required]
        public string Email { get; set; }
        [Required]
        public string Password { get; set; }
        [Required]
        public int CountryID { get; set; }
        [Required]
        public int PhoneNumber { get; set; }
        [Required]
        public string CPRNumber { get; set; }
        [Required]
        public string FirstName { get; set; }
        [Required]
        public string LastName { get; set; }
        [Required]
        public string Address { get; set; }
        [Required]
        public int ZipCodeID { get; set; }
        [Required]
        public int GenderID { get; set; }
    }
}

File: Website/Kasino/src/app/interfaces/CustomerRegisterRequest.ts
export interface CustomerRegisterRequest {
  email: string;
  password: string;
  countryID: number;
  phoneNumber: number;
  cprNumber: string;
  firstName: string;
  lastName: string;
  address: string;
  zipCodeID: number;
  genderID: number;
}
